app/components/ui/ReserveCard.tsx

```diff
@@ -27,57 +26,63 @@ export function ReserveCard({
 	addressLabel,
 	address,
 	tableData,
-	children,
 }: ReserveCardProps) {
+	const shouldShowTable = !isLoading && tableData && tableData.length > 0;
+
 	return (
-		<div className="card md:min-w-96">
-			<div className="card-body">
-				<p className="text-base-content/60 text-sm font-medium tracking-wide">{title}</p>
+		<div className="card md:min-h-54 md:min-w-96">
+			<div className="card-body justify-between">
+				<div className="flex flex-col gap-4">
+					<span className="text-base-content/60 text-sm font-medium tracking-wide">{title}</span>
 
-				{isLoading ? (
-					<div className="skeleton h-8 w-40" />
-				) : children ? (
-					children
-				) : (
-					<div className="flex items-center gap-3">
-						<p className="text-primary text-2xl font-bold sm:text-3xl">
-							{value} {unit}
-						</p>
-					</div>
-				)}
+					{isLoading ? (
+						<div className="skeleton h-8 w-40" />
+					) : (
+						<div className="flex items-center gap-3">
+							<p className="text-primary text-2xl font-bold sm:text-3xl">
+								{value} {unit}
+							</p>
+						</div>
+					)}
 
-				{tableData && (
-					<div className="mt-4 overflow-x-auto">
-						<table className="table-sm table">
-							<thead>
-								<tr>
-									<th className="text-base-content/75 font-medium">Name</th>
-									<th className="text-base-content/75 font-medium">Address</th>
-									<th className="text-base-content/75 text-right font-medium">Amount</th>
-								</tr>
-							</thead>
-							<tbody>
-								{tableData.map((item) => (
-									<tr key={item.address} className="hover:bg-base-200/50 border-base-300">
-										<td className="font-medium">{item.name}</td>
-										<td className="font-mono text-sm">{trimAddress(item.address)}</td>
-										<td className="text-right font-medium">
-											{item.amount} {item.unit}
-										</td>
+					{shouldShowTable && (
+						<div className="mt-4 overflow-x-auto">
+							<table className="table-sm table">
+								<thead>
+									<tr>
+										<th className="text-base-content/75 font-medium">Name</th>
+										<th className="text-base-content/75 font-medium">Address</th>
+										<th className="text-base-content/75 text-right font-medium">
```

Comment:
* why using custom color again?
* move repeted content higher up (to tr ot thead)